lightning/src/ln/chan_utils.rs

```diff
@@ -485,9 +485,11 @@ impl TxCreationKeys {
 }
 
 /// The maximum length of a script returned by get_revokeable_redeemscript.
-// Calculated as 6 bytes of opcodes, 1 byte push plus 2 bytes for contest_delay, and two public
-// keys of 33 bytes (+ 1 push).
-pub const REVOKEABLE_REDEEMSCRIPT_MAX_LENGTH: usize = 6 + 3 + 34*2;
+// Calculated as 6 bytes of opcodes, 1 byte push plus 3 bytes for contest_delay, and two public
+// keys of 33 bytes (+ 1 push). Generally, pushes are only 2 bytes (for values below 0x7fff, i.e.
+// around 7 months), however, a 7 month contest delay shouldn't result in being unable to reclaim
+// on-chain funds.
```

Comment:
Question:
I am not able to understand how 0x7fff corresponds to 7 months. Could you help me point to a resource that explains this? Thanks!

nit:

Since we are talking about more than a 7-month delay, which is also implied in the commit description, it makes sense to make it evident in the comment as well.

```suggestion
// keys of 33 bytes (+ 1 push). Generally, pushes are only 2 bytes (for values below 0x7fff, i.e.
// around 7 months), however, a more than 7-month contest delay shouldn't result in being unable to reclaim
// on-chain funds.
```

Reply:
Got it! Thanks for the reference @wpaulino